frontend/fingertips-frontend/app/results/page.tsx

```diff
@@ -99,6 +102,7 @@ export default async function Page(
         availableGroupTypes={availableGroupTypes}
         selectedAreasData={selectedAreasData}
         searchState={stateManager.getSearchState()}
+        serverDate={serverDate}
```

Comment:
Can you add a unit test to check this prop is passed as expected. The unit test for the page are already broken down by prop level. So should be a case of adding a new one rather than updating any existing one.